generators/generator-base.js

```diff
@@ -1927,3 +1927,21 @@ Generator.prototype.httpGet = function(url, onSuccess, onFail) {
         });
     }).on('error', onFail);
 };
+
+/**
+ * Function to print a proper array with simple quoted strings
+ *
+ *  @param {array} array - the array to print
+ */
+Generator.prototype.printProperArray = function(array) {
```

Comment:
rename to `toArrayString`
also instead of looping may be we could do ```array.join(`', '`)``` and then wrap the output in `[' ']` give it a try